subtitles.go

```diff
@@ -195,6 +244,8 @@ type StyleAttributes struct {
 	TeletextDoubleWidth  *bool
 	TeletextSpacesAfter  *int
 	TeletextSpacesBefore *int
+	TeletextVerticalPostion int
```

Comment:
Those 2 new attributes should be pointers so that we know when they've been initialized. You can use `astiptr.Int()` and `astiptr.Byte` from [astitools](https://github.com/asticode/go-astitools) package